Gemfile

```diff
@@ -27,6 +27,8 @@ gem 'devise', '~> 4.7'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'pry'
```

Comment:
Do you want to keep this in just development/test instead of also in production?

Reply:
Good call will move